setup.py

```diff
@@ -16,7 +17,7 @@
     license="BSD license",
     include_package_data=True,
     packages=find_packages(),
-    install_requires=["wagtail>=3.0,<5.0"],
+    install_requires=["wagtail>=4.1"],
```

Comment:
Was 4.1 an arbitrary choice, or are there specific modules / APIs in use that are no longer compatible with 3.0?

Reply:
Hi @vsalvino ,
The upcoming latest LTS will be Wagtail 5.2 and we usually retain support one LTS behind the latest one, which is 4.1.
The older versions were dropped because they have reached their end-of-life (EOL).

https://github.com/wagtail/wagtail/wiki/Release-schedule

Thanks for reviewing!